agent/pyproject.toml

```diff
@@ -49,6 +64,7 @@ build-backend = "poetry.core.masonry.api"
 
 [project.scripts]
 crystaldba = "crystaldba.cli.main:main"
+elia = "elia_chat.__main__:cli"
```

Comment:
`poetry run elia` does not work correctly for me. It takes the database connection string as input somehow.

`poetry run python -m elia_chat` does work.

Reply:
@jssmith do we want to do that until we have e2e tests working?